apps/prairielearn/src/pages/studentInstanceQuestion/studentInstanceQuestion.html.ts

```diff
@@ -34,184 +33,179 @@ export function StudentInstanceQuestion({
-          ? html`
-              ${compiledScriptTag('examTimeLimitCountdown.ts')}
-              ${EncodedData(
-                {
-                  serverRemainingMS: resLocals.assessment_instance_remaining_ms,
-                  serverTimeLimitMS: resLocals.assessment_instance_time_limit_ms,
-                  serverUpdateURL: `${resLocals.urlPrefix}/assessment_instance/${resLocals.assessment_instance.id}/time_remaining`,
-                  canTriggerFinish: resLocals.authz_result.authorized_edit,
-                  showsTimeoutWarning: true,
-                  reloadOnFail: true,
-                  csrfToken: resLocals.__csrf_token,
-                },
-                'time-limit-data',
-              )}
-            `
-          : ''}
-        <script defer src="${nodeModulesAssetPath('mathjax/es5/startup.js')}"></script>
-        <script>
-          document.urlPrefix = '${resLocals.urlPrefix}';
-        </script>
-        ${resLocals.variant == null
-          ? ''
-          : html`
-              ${resLocals.question.type !== 'Freeform'
-                ? html`
-                    <script src="${nodeModulesAssetPath('lodash/lodash.min.js')}"></script>
-                    <script src="${assetPath('javascripts/require.js')}"></script>
-                    <script src="${assetPath('localscripts/question.js')}"></script>
-                    <script src="${assetPath(
-                        `localscripts/question${resLocals.effectiveQuestionType}.js`,
-                      )}"></script>
-                  `
-                : ''}
-              ${unsafeHtml(resLocals.extraHeadersHtml)}
-            `}
-      </head>
-      <body>
-        ${Navbar({ resLocals })}
-        ${userCanDeleteAssessmentInstance
-          ? RegenerateInstanceModal({ csrfToken: resLocals.__csrf_token })
-          : ''}
-        <main id="content" class="container">
-          ${userCanDeleteAssessmentInstance ? RegenerateInstanceAlert() : ''}
-          <div class="row">
-            <div class="col-lg-9 col-sm-12">
-              ${resLocals.variant == null
-                ? html`
-                    <div class="card mb-4">
-                      <div class="card-header bg-primary text-white">
-                        <h1>
-                          ${QuestionTitle({
-                            questionContext,
-                            question: resLocals.question,
-                            questionNumber: resLocals.instance_question_info.question_number,
-                          })}
-                        </h1>
-                      </div>
-                      <div class="card-body">
-                        This question was not viewed while the assessment was open, so no variant
-                        was created.
-                      </div>
-                    </div>
-                  `
-                : QuestionContainer({ resLocals, questionContext, questionCopyTargets })}
-            </div>
-
```

Comment:
Why remove all these newlines? They were here to separate different sections.

Reply:
Fixed in [e1e23d4](https://github.com/PrairieLearn/PrairieLearn/pull/12376/commits/e1e23d46295272acd8e003c590636b6bd659def2).